### PR TITLE
sessionctx: show correct TiDB port instead of 3306

### LINKED ISSUE #9362: tidb-server always shows port 3306
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

```
mysql> show global variables like 'port';
```

2. What did you expect to see?

```
mysql> show global variables like 'port';
+---------------+-------+
| Variable_name | Value |
+---------------+-------+
| port          | 4000  |
+---------------+-------+
1 row in set (0.00 sec)
```

3. What did you see instead?
```

mysql> show global variables like 'port';
+---------------+-------+
| Variable_name | Value |
+---------------+-------+
| port          | 3306  |
+---------------+-------+
1 row in set (0.00 sec)
```

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
```
mysql> select tidb_version()\G
*************************** 1. row ***************************
tidb_version(): Release Version: v3.0.0-beta-55-gb9c5f55cc
Git Commit Hash: b9c5f55cc0259481734a204a9190a36f0761beea
Git Branch: accurate-port
UTC Build Time: 2019-02-19 06:48:36
GoVersion: go version go1.11.4 linux/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
1 row in set (0.00 sec)
```

### PR DESCRIPTION
**Pull request by morgo**
### What problem does this PR solve? 

Fixes #9362

### What is changed and how it works?

Previously TiDB lied about the port configured (showing MySQL's default of 3306).

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has exported variable/fields change

Side effects

 - Increased code complexity
 - Breaking backward compatibility (for broken apps)

Related changes

 - Need to be included in the release note

### CODE REVIEW

**Comment by morgo** [comment]
@jackysp PTAL

File: sessionctx/variable/sysvar.go [hunk 1]
```diff
@@ -363,7 +363,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal, "innodb_buffer_pool_load_now", "OFF"},
 	{ScopeNone, "performance_schema_max_rwlock_classes", "40"},
 	{ScopeNone, "binlog_gtid_simple_recovery", "OFF"},
-	{ScopeNone, "port", "3306"},
+	{ScopeNone, "port", strconv.FormatUint(uint64(config.GetGlobalConfig().Port), 10)},
```
Comment (zz-jason) [inline sessionctx/variable/sysvar.go]:
```suggestion
	{ScopeNone, Port, strconv.FormatUint(uint64(config.GetGlobalConfig().Port), 10)},
```

**Pull request merged**